README.md

```diff
@@ -979,6 +979,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 * [bluss/ndarray](https://github.com/rust-ndarray/ndarray) — N-dimensional array with array views, multidimensional slicing, and efficient operations
 * [kernelmachine/utah](https://github.com/kernelmachine/utah) — Dataframe structure and operations in Rust
+* [ritchie46/polars](https://github.com/ritchie46/polars) - Fast feature complete DataFrame library [![Build Status](https://travis-ci.com/ritchie46/polars.svg?branch=master)](https://travis-ci.com/ritchie46/polars)
```

Comment:
The build badge URL 301's to https://api.travis-ci.com/ritchie46/polars.svg?branch=master which is what's breaking the build. Can you fix that please?

Reply:
Right! Fixed.